packages/consumption/src/modules/requests/outgoing/OutgoingRequestsController.ts

```diff
@@ -115,7 +115,11 @@ export class OutgoingRequestsController extends ConsumptionBaseController {
     private async _create(id: CoreId, content: Request, peer: CoreAddress) {
         const canCreateResult = await this.canCreate({ content, peer });
 
-        if (canCreateResult.isError()) throw canCreateResult.error;
+        if (canCreateResult.isError()) {
+            const error = ConsumptionCoreErrors.requests.inheritedFromItem("Some child items have errors. Call 'canCreate' to get more information.");
+            if (canCreateResult.error.equals(error)) throw error;
+            throw canCreateResult.error;
```

Comment:
```suggestion
            if (canCreateResult.error.equals(error)) throw error;

            throw canCreateResult.error;
```

as this results in very different things I'd love to see a newline here to make this easier to process.